docs/commands/index.md

```diff
@@ -5,22 +5,19 @@ tocMaxDepth: 2
 
 Redocly OpenAPI CLI currently supports the following commands:
 
-- [bundle](bundle.md)
-- [join](join.md)
-- [lint](lint.md)
-- [login](login.md)
-- [logout](logout.md)
-- [preview-docs](preview-docs.md)
-- [push](push.md)
-- [split](split.md)
-- [stats](stats.md)
+* [bundle](bundle.md)
+* [join](join.md)
+* [lint](lint.md) 
+* [login](login.md)
+* [logout](logout.md)
+* [preview-docs](preview-docs.md)
+* [push](push.md)
+* [split](split.md)
+* [stats](stats.md)
 
+To define the behavior of the commands, you can use configuration files:
 
-The following configuration files define the behavior of the CLI tool:
+- `.redocly.yaml` - define the location of your root files, linting rules, and reference docs configuration information.
+- `.redocly.lint-ignore.yaml` - ignore specific lint messages.
 
-- `.redocly.yaml` - used to define the location of your root files, linting rules, and reference docs configuration information.
-- `.redocly.lint-ignore.yaml` - used to ignore specific lint messages.
-
-The CLI tool looks for configuration files in the current working directory. If it detects them, it will use the options set in those configuration files for the commands. Learn more about the [configuration file structure and options](../configuration/index.mdx).
-
-When executing any of the commands, you can override the default configuration file by providing a path to another configuration file with the `--config` option.
+For more information about configuration, please refer to the [corresponding](../configuration/index.mdx) section.
```

Comment:
```suggestion
For more information, refer to the [configuration section](../configuration/index.mdx).
```

Reply:
This suggestion is just to simplify the sentence a little bit, and to make the anchor text more relevant. In our docs, we try to avoid "empty", generic words and phrases like "here" , "this",  "related" as anchor text, and instead make it more descriptive whenever possible. This is supposedly good for SEO :) but it also helps the readers know where exactly the link will take them.